chainerx_cc/chainerx/backward.cc

```diff
@@ -621,13 +639,27 @@ std::vector<absl::optional<Array>> Grad(
         }
     }
 
-    BackwardImpl{inputs, outputs, actual_backprop_id, double_backprop, std::move(array_node_grad_map)}.Run();
+    // Push initial output grads, Run assigns to 1 in other case
```

Comment:
A nitpick: to more easily understand that it refers to a function.

```suggestion
    // Push initial output grads, Run() assigns to 1 in other case
```